deploy/helm/jumpstarter/charts/jumpstarter-controller/model.py

```
@@ -0,0 +1,278 @@
+        None,
+        description="Whether to allow keepalive pings even when there are no active streams(RPCs)",
+    )
+
+
+class Grpc(BaseModel):
+    model_config = ConfigDict(extra="forbid")
+
+    keepalive: Optional[Keepalive] = None
+
+
+class Metrics(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)